### PR TITLE
Add CAST AI Cluster token resource

### PR DESCRIPTION
**Pull request by matas-cast**

### CODE REVIEW

File: castai/resource_castai_eks_cluster.go [hunk 1]
```diff
@@ -279,6 +274,7 @@ func resourceCastaiEKSClusterDelete(ctx context.Context, data *schema.ResourceDa
 			log.Printf("[INFO] Disconnecting cluster.")
 			response, err := client.ExternalClusterAPIDisconnectClusterWithResponse(ctx, clusterId, sdk.ExternalClusterAPIDisconnectClusterJSONRequestBody{
 				DeleteProvisionedNodes: getOptionalBool(data, FieldEKSClusterDeleteNodesOnDisconnect, false),
+				KeepKubernetesResources: toBoolPtr(true),
```
Comment (aldor007) [inline castai/resource_castai_eks_cluster.go:277]: Can you add this for GKE also?

Comment (matas-cast) [reply castai/resource_castai_eks_cluster.go:277]: sure

File: castai/resource_castai_eks_cluster.go [hunk 2]
```diff
@@ -170,11 +170,6 @@ func resourceCastaiEKSClusterCreate(ctx context.Context, data *schema.ResourceDa
 	}
 
 	clusterID := *resp.JSON200.Id
-	tkn, err := createClusterToken(ctx, client, clusterID)
-	if err != nil {
-		return diag.FromErr(err)
-	}
-	data.Set(FieldEKSClusterToken, tkn)
```
Comment (aldor007) [inline castai/resource_castai_eks_cluster.go:177]: IMO we whould remove `FieldEKSClusterToken` from schema also. Now it will be an empty string

Comment (matas-cast) [reply castai/resource_castai_eks_cluster.go:177]: Of course, good 👁️

**Review by aldor007 — APPROVED** [review]

**Pull request merged**